net-misc/ipmiview/ipmiview-2.18.0.201007.ebuild

```diff
@@ -20,7 +20,7 @@ KEYWORDS="-* ~amd64"
 RDEPEND="
 	net-misc/stunnel
 	sys-libs/ncurses-compat:5
-	virtual/jre:1.8
+	>=virtual/jre-1.8:*
```

Comment:
this needs a revbump as it changes the information stored in `package.env`.

Reply:
Makes sense, I'll fix it.

Reply:
done.